notebooks/instructlab-knowledge/instructlab-knowledge.ipynb

```diff
@@ -270,8 +272,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#print(all_chunks)\n",
-    "print(chunks[0])"
+    "import random\n",
+    "import json\n",
+    "\n",
+    "sample = random.sample(all_chunks, 5)\n",
```

Comment:
This will raise and error if there are less than 5 chunks, maybe do something like

```suggestion
    "sample = random.sample(all_chunks, min(len(all_chunks), 5))\n",
```